pages/positions/position.tsx

```diff
@@ -0,0 +1,55 @@
+import type {NextPage} from 'next'
+import {usePositionHandler} from '../../hooks/usePositionHandler'
+
+type PositionProps = {
+  positions: {asset: string; amountOfLiquidity: string; amountOfCollateral: string; profit: string; loss: string}[] | undefined
+}
```

Comment:
prettify this, try not to structure objects in a linear fashion

Reply:
Usama, if this is not in prettier config can you put it? thank you.

Reply:
Ok